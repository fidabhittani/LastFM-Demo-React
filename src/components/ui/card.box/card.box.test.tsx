import React from "react";
import { shallow } from "enzyme";
import CardBox from "./index";

/** CardBox TEST */
describe("CardBox Root Component Tests", () => {
  it("CardBox Component renders correctly", () => {
    const cardBox = shallow(<CardBox />);
    expect(cardBox).toMatchSnapshot();
  });
});
