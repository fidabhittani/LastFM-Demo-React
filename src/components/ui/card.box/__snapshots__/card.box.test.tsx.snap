// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBox Root Component Tests CardBox Component renders correctly 1`] = `ShallowWrapper {}`;
